include/boost/geometry/algorithms/detail/equals/implementation.hpp

```diff
@@ -275,6 +275,15 @@ struct equals_by_collection_or_relate
     }
 };
 
+struct equals_always_false
+{
+    template <typename Geometry1, typename Geometry2, typename Strategy>
+    static inline bool apply(Geometry1 const& , Geometry2 const& , Strategy const& )
```

Comment:
Nitpicking: `& )` -> `&)` :-)

Reply:
@mloskot Well idk ;) do we have gudelines for that? AFAIU we put space after `&` and before `, ` so I assumed that the missing name is represented by space. Or am I mistaken and all of the spaces should be removed?

Reply:
AFAIK we don't have a written guideline for that.
I find this readable, but I'm also fine with &)

Reply:
Having nothing constructive to say, I went nit-picking and bike-shedding. Certainly, ignoreable

Reply:
Certainly nice to hear from you Mateusz!

Reply:
Barend, I'm happy to be revolving around still

Reply:
Ok, I saw this in Geometry code but it is possible that it was me who put it in the first place. I don't have an opinion regarding it, but if it's ignorable I'll leave it as it is. ;)